tools/cd_scripts/e2e_test.sh

```diff
@@ -71,6 +71,11 @@ then
     #install git
     sudo apt install -y git
 
+    # Downloading composite object requires integrity checking with CRC32c in gsutil.
+    # it requires to install crcmod.
+    sudo apt-get install -y gcc python3-dev python3-setuptools
+    sudo apt install -y python3-crcmod
```

Comment:
```
CentOS, RHEL, and Fedora
  ------------------------

  To compile and install crcmod:

    yum install gcc python3-devel python3-setuptools redhat-rpm-config
    sudo pip3 uninstall crcmod
    sudo pip3 install --no-cache-dir -U crcmod

  Debian and Ubuntu
  -----------------

  To compile and install crcmod:

    sudo apt-get install gcc python3-dev python3-setuptools
    sudo pip3 uninstall crcmod
    sudo pip3 install --no-cache-dir -U crcmod
```

Why are we not following the installation method provided by `gsutil help crcmod` command for Debian and Ubuntu?

Reply:
It is getting installed with python3 library, so why should we do extra effort for requirements and all.

Reply:
this is alternative of pip3 only for debian and ubuntu.